mms/metrics/system_metrics.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is this file only used in the example?

Reply:
this is used by the front end to collect system level metrics